docs/getting-started/accessing-resources/getting-access-to-data.md

```diff
@@ -103,20 +137,147 @@ To list all available releases, you can use the following command from the root
 The release directory is symlinked to `data/current`.
 This is generally the path you should use in your code.
 
-## Accessing simulated data
 
-You can also use the download data script to download the test data for the project by using the `--test-data` option.
-The test data are generally smaller, simulated or permuted data files with the same structure as the real project data (learn more at the [`simulate-sce`](https://github.com/AlexsLemonade/OpenScPCA-analysis/tree/main/analyses/simulate-sce) module).
+## Accessing ScPCA module results
+
+As an analysis modules matures, the Data Lab will port the analysis to a separate repository called [`OpenScPCA-nf`](https://github.com/AlexsLemonade/OpenScPCA-nf).
+This repository holds a [Nextflow workflow](https://www.nextflow.io/) to reproducibly run and generate results from completed analysis modules.
+
+If you wish to use output from a completed analysis module in your module, you will need to obtain the results generated by this workflow.
+
+We have provided a [`download-results.py` script](https://github.com/AlexsLemonade/OpenScPCA-analysis/blob/main/download-results.py) that you can use to download results from a module that has been run in `OpenScPCA-nf`.
+
+We briefly cover some of the most common use cases below, but we encourage you to review all the options available to you.
+
+You can list all the options for `download-results.py` by running the following from the root directory of the repository:
+
+```sh
+./download-results.py --help
+```
+
+!!! tip "Log into AWS CLI before running the script"
+    Before running this script locally, you will need to be [logged into your AWS CLI profile](../../technical-setup/environment-setup/configure-aws-cli.md#logging-in-to-a-new-session).
+    To do this, run the following commands in terminal before running the `results-download.py` script, and follow instructions to log in.
+
+    ```sh
+    # replace `openscpca` with your AWS CLI profile name if it differs
+    export AWS_PROFILE=openscpca
+    aws sso login
+    ```
+
+    The command `export AWS_PROFILE=openscpca` will define your AWS profile name as `openscpca` for the duration of your terminal session.
+    Defining this variable is helpful because the `download-data.py` script also needs your profile name to download data from S3.
+
+    Note that you can also [add this profile definition to your shell profile file](../../technical-setup/environment-setup/configure-aws-cli.md#storing-your-aws-profile-name) so that it will always be defined in any terminal session.
+Example script usage below assumes you have created a profile called `openscpca` when [configuring the AWS CLI](../../technical-setup/environment-setup/configure-aws-cli.md).
+
+To download all results for one or more modules, use the `--modules` option as follows:
```

Comment:
Since later we will say how to avoid all, let's just leave this a bit ambiguous?
```suggestion
To download results for one or more modules, use the `--modules` option as follows:
```